StardewArchipelago/Locations/CodeInjections/Modded/SVE/BearShopStockModifier.cs

```diff
@@ -88,7 +88,8 @@ private double BearDiscount()
             {
                 applesHearts = Game1.player.friendshipData["Apples"].Points / 250; // Get discount from being friends with Apples
             }
-            return 1 - INITIAL_DISCOUNT - (knowledgeBuff + applesHearts * APPLES_DISCOUNT);
+            var additionalDiscount = knowledgeBuff + applesHearts * APPLES_DISCOUNT;

```

Comment:
These 3 values are still 0.7 + 0.4 + (0.025 * 10) which can total 1.35, which is a discount of 135%